make_site.py

```diff
@@ -173,6 +174,18 @@ class HundredTheorem:
     links: Optional[Mapping[str, str]] = None
     note: Optional[str] = None
 
+@dataclass
+class ThousandPlusTheorem:
+    # Wikidata identifier: the letter Q followed by a string as digits, e.g. "Q1008566"
+    wikidata: str
```

Comment:
One thing, just because to be precise: this is the identifier *with* an additional `X`/`A`/`B` appended, right? So this is not technically the Wikidata identifier, but a 1000+-theorems identifier. Could you be specific in the comment at least?

Reply:
Good catch, fixed!